chatbot-core/backend/app/models/chat.py

```diff
@@ -323,3 +326,44 @@ class ChatSessionResponse(BaseModel):
 
     class Config:
         from_attributes = True
+
+
+class ChatFeedback(Base):
+    """
+    Represents feedback for a chat message.
+    Tracks user feedback, rating, and associated metadata.
+    """
+
+    __tablename__ = "chat_feedback"
+
+    id: Mapped[UNIQUEIDENTIFIER] = mapped_column(
+        UNIQUEIDENTIFIER(as_uuid=True), primary_key=True, default=uuid4
+    )
+    chat_message_id: Mapped[UNIQUEIDENTIFIER] = mapped_column(
+        ForeignKey("chat_message.id", ondelete="CASCADE"), nullable=False
+    )
+    is_positive: Mapped[bool] = mapped_column(Boolean, nullable=False)
+    feedback_text: Mapped[Optional[str]] = mapped_column(Text, nullable=True)
+    created_at: Mapped[datetime] = mapped_column(
```

Comment:
Where is the field `predefined_feedback: Mapped[str | None] = mapped_column(String, nullable=True)` that exists in the Danswer original code ?